sales_tax.rb

```diff
@@ -0,0 +1,68 @@
+class Products
```

Comment:
@mohseengs for making these object oriented you could have made specific attributes related to products in product class and methods also. And same for tax tax could have contained fields and methods related to tax